djangocms_icon/templates/admin/djangocms_icon/includes/assets.html

```diff
@@ -1 +1 @@
-<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
```

Comment:
What is the difference between 5.3.1 and the latest visible 5.7.2:
https://fontawesome.com/start

Reply:
The Bootstrap iconpicker only supports up to version 5.3.1. Loading 5.7.2 would not be consistent in my opinion.